doc/fropin.xml

```diff
@@ -183,27 +183,34 @@ true]]></Example>
   </ManSection>
 <#/GAPDoc>
 
-<#GAPDoc Label="RightCayleyGraphSemigroup">
+<#GAPDoc Label="RightCayleyDigraph">
   <ManSection>
-    <Attr Name="RightCayleyGraphSemigroup" Arg="S"/> 
-    <Attr Name="LeftCayleyGraphSemigroup" Arg="S"/> 
+    <Attr Name="RightCayleyDigraph" Arg="S"/> 
+    <Attr Name="LeftCayleyDigraph" Arg="S"/> 
     <Returns>A list of lists of positive integers.</Returns>
     <Description>
       When the argument <A>S</A> is a semigroup in the representation 
       <Ref Filt="IsEnumerableSemigroupRep"/>, 
-      <C>RightCayleyGraphSemigroup</C> returns the right
-      Cayley graphs of <A>S</A>, as a list <C>graph</C> where 
-      <C>graph[i][j]</C> is equal to 
+      <C>RightCayleyDigraph</C> returns the right
+      Cayley graphs of <A>S</A>, as a <Ref Oper="Digraph" BookName="digraphs"/>
+      <C>digraph</C> where vertex <C>OutNeighbours(digraph)[i][j]</C> is 
       <C>PositionCanonical(<A>S</A>, AsListCanonical(<A>S</A>)[i] *
         GeneratorsOfSemigroup(<A>S</A>)[j])</C>.
-      The list returned by <C>LeftCayleyGraphSemigroup</C> is defined analogously.
+      The list returned by <C>LeftCayleyDigraph</C> is defined analogously.<P/>
```

Comment:
This sentence implies that `<C>LeftCayleyDigraph</C>` returns a list, rather than a digraph.